xml/ftp_banners.xml

```diff
@@ -129,15 +129,24 @@ example.com FTP server (Version:  Mac OS X Server) ready.</example>
     <param pos="2" name="service.version"/>
     <param pos="1" name="host.name"/>
   </fingerprint>
-  <fingerprint pattern="^(\S+) FTP Server \(SunOS (\S+)\) ready\.?$" flags="REG_ICASE">
+  <fingerprint pattern="^(\S+) FTP Server \(SunOS 5.([789]|\d{2,})\) ready\.?$" flags="REG_ICASE">
     <description>SunOS/Solaris</description>
-    <example>example.com FTP server (SunOS 5.7) ready.</example>
+    <example host.name="example.com" os.version="7">example.com FTP server (SunOS 5.7) ready.</example>
```

Comment:
Should probably add a check to verify that double digit versions match OK. 
SunOS 5.11 -> Solaris 11